style.css

```diff
@@ -55,26 +124,211 @@ h1 {
   color: rgba(23, 43, 77, 1);
 }
 
-.software {
-  font-size: 16.3px;
+.ulto {
+  margin-top: 35px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 236px;
+  height: 40px;
+  order: 1;
+  flex-grow: 0;
+}
+
+.ulto > li {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  margin-left: 10px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.daily {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
   line-height: 24px;
   color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
 }
 
-.connect {
-  font-size: 16px;
-  margin-top: 15px;
+.flex-box {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
   line-height: 24px;
-  color: #7f8cff;
+  width: 203px;
+  height: 50px;
+  font-size: 12px;
+  color: #6070ff;
+}
+
+.flex-box .box1 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 5px 12px;
+  width: 48px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.flex-box .box2 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 2px 12px;
+  width: 42px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.flex-box .box3 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 2px 12px;
+  width: 84px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+}
+
+.but {
+  background-color: white;
+  font-style: normal;
   font-weight: 500;
+  font-size: 17px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  width: 120px;
+  height: 48px;
+  margin-top: 15px;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
```

Comment:
- [ ]  Great working implementing the figma design, however your button is still missing hover, active and pressed status as per the project requirement.

The button (e.g., "Get my resume") is a link a or button tag, and follow the button styling: Active, Hover and Pressed (see design)